client/apps/queue/routes.coffee

```diff
@@ -0,0 +1,27 @@
+_ = require 'underscore'
+query = require './query.coffee'
+Lokka = require('lokka').Lokka
+Transport = require('lokka-transport-http').Transport
+{ API_URL } = process.env
+
+@queue = (req, res, next) ->
+  published = true
+  channel_id = req.user?.get('current_channel').id
+  queuedQuery = query "daily_email: true"
+  latestQuery = query "channel_id: \"#{channel_id}\", published: true, sort: \"-published_at\", daily_email: false"
+
+  client = new Lokka
+    transport: new Transport(API_URL + '/graphql')
+
+  client.query(queuedQuery).then (result) =>
+    if result.error
+      return res.backboneError
```

Comment:
I don't think you want to use `res.backboneError` here. Since this is now in GraphQL land we don't need that helper. You could use next here, e.g. `return next result.error if result.error` or more likely since Lokka uses promises you would just catch the errors all at once like...

````coffeescript
client.query(...)
  .then (result) =>
    # ...
  .catch(next)
````